test.js

```diff
@@ -218,7 +218,7 @@ it('function constructor', function() {
     'throw new Function(")");'
   ], [
     'SyntaxError: Unexpected token )',
-    /^    at Object\.Function \((?:unknown source|<anonymous>)\)$/,
+    /^    at Function \((?:unknown source|<anonymous>|native)\)$/,
```

Comment:
I think this change may be necessary due to some change in Node 0.12 and/or io.js, am I right?
If so, it should be move in its own PR.

Also, simply removing the `Object\.` prefix may be breaking the tests on some platforms.